libphobos/src/std/math.d

```diff
@@ -2258,16 +2282,16 @@ unittest
         version (IeeeFlagsSupport)
         {
             // Check the overflow bit
-            //if (x == real.infinity)
-            //{
-            //    // don't care about the overflow bit if input was inf
-            //    // (e.g., the LLVM intrinsic doesn't set it on Linux x86_64)
-            //    assert(pair[0] == real.infinity || f.overflow);
-            //}
-            //else
-            //    assert(!f.overflow);
+            if (x == real.infinity)
+            {
+                // don't care about the overflow bit if input was inf
+                // (e.g., the LLVM intrinsic doesn't set it on Linux x86_64)
+                assert(pair[0] == real.infinity || f.overflow);
+            }
+            else
+                assert(!f.overflow);
             // Check the underflow bit
-            //assert(f.underflow == (fabs(x) < real.min_normal));
+            assert(f.underflow == (fabs(x) < real.min_normal));
```

Comment:
Ieee flag check has been turned on, as the generic math paths now at least _try_ to raise an fp exception, e.g:
```
return real.min_normal * copysign(real.min_normal, 0.0);
```